src/utils/update/company.profile.mapper.to.overseas.entity.ts

```diff
@@ -1,18 +1,19 @@
 import { CompanyProfile } from "@companieshouse/api-sdk-node/dist/services/company-profile/types";
 import { yesNoResponse } from "../../model/data.types.model";
 import { Entity } from "../../model/entity.model";
-import { isSameAddress, mapAddress } from "../../utils/update/mapper.utils";
+import { isSameAddress, lowerCaseAllWordsExceptFirstLetters, mapAddress } from "../../utils/update/mapper.utils";
 
 export const mapCompanyProfileToOverseasEntity = (cp: CompanyProfile): Entity => {
   const serviceAddress = mapAddress(cp.serviceAddress);
   const principalAddress = mapAddress(cp.registeredOfficeAddress);
+
   return {
     registration_number: cp.foreignCompanyDetails?.registrationNumber,
     law_governed: cp.foreignCompanyDetails?.governedBy,
     legal_form: cp.foreignCompanyDetails?.legalForm,
-    incorporation_country: cp.jurisdiction,
+    incorporation_country: lowerCaseAllWordsExceptFirstLetters(cp.foreignCompanyDetails?.originatingRegistry?.country as string),
```

Comment:
handle undefined (don't cast "as string" in case it's undefined)

Reply:
removed 'as string' part and added 'string | undefined' to param type